frontend/src/app/main/ui/ds/notifications/actionable.scss

```diff
@@ -0,0 +1,26 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) KALEIDOS INC
+
+@use "../_borders.scss" as *;
+
+.notification {
+  align-items: center;
+  background: var(--color-background-primary);
+  border-radius: $br-8;
+  border: $b-1 solid var(--color-background-quaternary);
+  display: flex;
+  justify-content: space-between;
+  padding: var(--sp-l);
+
+  :global(a) {
+    color: var(--color-accent-primary);
+  }
```

Comment:
Please let's not use global styles

Reply:
How can I style the text that is a children otherwise? the text that it's added as a child is from markdown so we cannot customize its class.